testutil/integration/grpc/grpc.go

```diff
@@ -0,0 +1,52 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+package grpc
+
+import (
+	sdktypes "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/evmos/evmos/v14/testutil/integration/network"
+	evmtypes "github.com/evmos/evmos/v14/x/evm/types"
+	feemarkettypes "github.com/evmos/evmos/v14/x/feemarket/types"
+	revtypes "github.com/evmos/evmos/v14/x/revenue/v1/types"
+)
+
+// Handler is an interface that defines the methods that are used to query
+// the network's modules via gRPC.
+type Handler interface {
+	// EVM methods
+	GetEvmAccount(address common.Address) (*evmtypes.QueryAccountResponse, error)
+	EstimateGas(args []byte, GasCap uint64) (*evmtypes.EstimateGasResponse, error)
+	GetEvmParams() (*evmtypes.QueryParamsResponse, error)
+
+	// Bank methods
+	GetBalance(address sdktypes.AccAddress, denom string) (*banktypes.QueryBalanceResponse, error)
+
+	// Account methods
+	GetAccount(address string) (authtypes.AccountI, error)
+
+	// FeeMarket methods
+	GetBaseFee() (*feemarkettypes.QueryBaseFeeResponse, error)
+
+	// Revenue methods
+	GetRevenue(address common.Address) (*revtypes.QueryRevenueResponse, error)
+	GetRevenueParams() (*revtypes.QueryParamsResponse, error)
+}
+
+var _ Handler = (*IntegrationHandler)(nil)
+
+// IntegrationHandler is a helper struct to query the network's modules
+// via gRPC. This is to simulate the behavior of a real user and avoid querying
+// the modules directly.
+type IntegrationHandler struct {
+	network network.Network
+}
+
+// NewGrpcHandler creates a new IntegrationGrpcHandler instance.
+func NewGrpcHandler(network network.Network) Handler {
```

Comment:
nit: abbreviations should be capitalized

```suggestion
func NewGRPCHandler(network network.Network) Handler {

```